atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/ApiTestSchema.java

```diff
@@ -88,6 +89,9 @@ private static Schema generateSchema() {
             }
         });
 
+        schema.addStreamStoreDefinition(
+                new StreamStoreDefinitionBuilder("stream_test", "stream_test", ValueType.VAR_LONG).build());
```

Comment:
We didn't have any tests for the stream store renderer, so I've added a stream store to the test schema.